pages/common/nuclei.md

```diff
@@ -0,0 +1,36 @@
+# nuclei
+
+> Fast and customizable vulnerability scanner based on simple YAML based DSL.
+> More information: <https://github.com/projectdiscovery/nuclei>.
+
+- Update nuclei templates to the latest released version:
+
+`nuclei -ut`
+
+- List all templates (of protocol type dns):
+
+`nuclei -tl {{-pt dns}}`
+
+- Run automatic web scan using wappalyzer technology detection:
+
+`nuclei -as -u {{scanme.nmap.org}}`
+
+- Run http protocol type templates of high and critical severity; write results to markdown files:
+
+`nuclei -severity high,critical -pt http -u {{http://scanme.sh}} -me {{markdown_dir}}`
+
+- Run all templates using rate limiting options with reduced silent output:
+
+`nuclei -rl {{150}} -bs {{25}} -c {{25}} -silent -u {{http://scanme.sh}}`
+
+- Run (wordpress) [w]orkflow against a (wordpress) site:
+
+`nuclei -w {{path/to/nuclei-templates/workflows/wordpress-workflow.yaml}} -u {{https://sample.wordpress.site}}`
+
+- Run specific template collections from a directory with verbose output in STDERR and detected issues/vulns to a file:
+
+`nuclei -t {{path/to/nuclei-templates/http}} -u {{http://scanme.sh}} -v -o {{results}}`
```

Comment:
```suggestion
- Run one or more specific [t]emplate or directory with [t]emplates with [v]erbose output in `stderr` and [o]utput detected issues/vulns to a file:

`nuclei -t {{path/to/nuclei-templates/http}} -u {{http://scanme.sh}} -v -o {{results}}`
```
I would like to highlight 2 points here:
1. IMO collection sounds like a somewhat special predefined set of templates, which from my interpretation of the README, it only means all templates in the directory:
>    -t, -templates string[]                list of template or template directory to run (comma-separated, file)
2. We use standard notation for some techical terms, which is the case of standard streams, which must be spelled in the following way: `stdout`, `stderr`, `stdin`. See the [style guide](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md#special-cases):
> ### Special cases
> 
> - If a command performs irreversible changes to a file system or devices,
>   write every example in a way that cannot be copy pasted thoughtlessly.
>   For example, instead of `ddrescue --force --no-scrape /dev/sda /dev/sdb`
>   write `ddrescue --force --no-scrape {{/dev/sdX}} {{/dev/sdY}}`
>   and use the `{{/dev/sdXY}}` placeholder for *block devices* instead of `/dev/sda1`.
> 
> In general, placeholders should make it as intuitive as possible
> to figure out how to use the command and fill it in with values.
> 
> Technical wording on description lines should use the `backtick` syntax.
> Use backticks on the following:
> 
> - Paths, e.g. `package.json`, `/etc/package.json`.
> - Extensions, e.g. `.dll`.
> - Commands, e.g. `ls`.
> - Standard streams: `stdout`, `stdin`, `stderr`. **Do not** use the full names (e.g. standard output).